src/sql/parts/profiler/editor/controller/profilerTableEditor.ts

```diff
@@ -237,7 +263,24 @@ export class ProfilerTableEditor extends BaseEditor implements IProfilerControll
 		}
 	}
 
+	private _updateRowCountStatus(): void {
+		if (this._showStatusBarItem) {
+			let message = this._input.data.filterEnabled ? `${RowCountFilteredStatusBarText} ${this._input.data.getLength()}/${this._input.data.getLengthNonFiltered()}` : `${RowCountStatusBarText} ${this._input.data.getLength()}`;
```

Comment:
This won't work for localization since not all translations will necessarily put the number at the end of the text. The right way to do it is include a parameter in the string and then pass an argument to the `localize` call with the parameter's value. See https://github.com/Microsoft/vscode-nls for docs, there's an example in the first code block in the readme